build/release/README.md

```diff
@@ -79,6 +79,12 @@ After the PR is merged, you can tag the release with the beta tag (`v1.13.0-beta
 
 **IMPORTANT** Before tagging the release, open a new PR to update the documentation and example manifest tags to the release version.
 
+The script  `set-release-ver.sh` can help preparing these changes:
+
+```console
+# Pass the new version
+$ tests/scripts/set-release-ver.sh v1.16.8
```

Comment:
Other console snippets above don't use dollar signs. If we leave this out copying and pasting is easier with less friction

```suggestion
tests/scripts/set-release-ver.sh v1.16.8
```